src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,18 +24,32 @@ const (
     // Always begin a "tag" with a dash (as per http://semver.org)
     tag   = "-beta"
```

Comment:
Ah, I'll actually just go ahead and delete the comment all together. However you are right, I may as well delete the dash in the beginning here.